cdap-client/src/main/java/co/cask/cdap/client/AuthorizationClient.java

```diff
@@ -80,11 +81,21 @@ public AuthorizationClient(ClientConfig config) {
 
   @Override
   public void enforce(EntityId entity, Principal principal, Action action) throws Exception {
+    enforce(entity, principal, Collections.singleton(action));
+  }
+
+  @Override
+  public void enforce(EntityId entity, Principal principal, Set<Action> actions) throws Exception {
     throw new UnsupportedOperationException("Enforcement is not supported via Java Client. Please instead use the " +
```

Comment:
the above method just calls this method. So it will throw Unsupported as well :)